libaftermath-core/src/defs/aftermath/types/openmp/on_disk.py

```diff
@@ -238,18 +238,6 @@
 
 ################################################################################
 
-am_dsk_openmp_flush = EventFrame(
-    name = "am_dsk_openmp_flush",
-    entity = "on-disk OpenMP flush event",
-    comment = "Memory flush event triggered by OpenMP flush constructs",
-    fields = FieldList([
-        Field(
-            name = "timestamp",
-            field_type = aftermath.types.builtin.uint64_t,
-            comment = "Time of the event")]))
-
-#################################################################################
-
```

Comment:
It seems that multiple commits got mixed up in this one. Maybe split the commit? If previously introduced data structures are removed, please remove them in the original commit.

Reply:
Resolved after commits got squashed.